package.json

```diff
@@ -50,6 +51,7 @@
     "highcharts": "9.1.0",
     "highcharts-react-official": "3.0.0",
     "keycloak-js": "^15.0.2",
+    "lodash.truncate": "^4.4.2",
```

Comment:
It's a bit strange to include only one fn out of lodash

let's roll with the full support for `lodash` instead

Reply:
https://github.com/podkrepi-bg/frontend/pull/719/commits/15d0d06b172ace6de860f1dc6336566ff4dad008